cmd/client.go

```diff
@@ -32,6 +33,7 @@ var (
 	dlaasGrpcClient TrainerClient //Only initiate if we are using gRPC (initiate via trainModelGrpc())
 	defaultBackoff  = backoff.NewExponentialBackOff()
 	gRPCpath        = os.Getenv("DLAAS_GRPC")
+	gRpcCertString  = os.Getenv("FFDL_GRPC_CERT") // ca.crt
```

Comment:
Why is reading from an environment better than a file?  Seems to me a file is better, but I'm willing to be corrected.

Reply:
It's slightly easier in the instructions as well, since we already need to set environment variables. And I'm not sure we can rely on the trainer being available in case FfDL was setup via helm charts, so `ffdl-trainer/envs/dev_values.yaml` is not guaranteed to exist.

Reply:
Side note: You can always run something like `export FFDL_GRPC_CERT=$(python -c "import yaml;from pathlib import Path;test=yaml.safe_load(Path(\"${HOME}/go/src/github.com/AISphere/ffdl-trainer/envs/dev_values.yaml\").read_text());print(test['certs']['ca_crt'])")` if you want to read from a specific file.